WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -20,6 +20,50 @@ public class ReaderScreen: ScreenObject {
         )
     }
 
+    public func openTheLastPostInApp() {
```

Comment:
Nitpick: Function names don't usually have articles in them. I don't see it spelled out clearly in the [Swift API Guidelines](https://www.swift.org/documentation/api-design-guidelines/#naming) and, thinking about it, I never really read this anywhere, but, based on all the code I've read over time, I think my assertion is accurate.

```suggestion
    public func openLastPostInApp() {
```

Also, I think it's safe to omit "in app" from the name. I'm guessing you did it to differentiate it from the method below that uses Safari. But, IMHO, if nothing else is specified it's safe to assume the action will occur in the app.

Reply:
Yeah, the idea was really to differentiate it. I was actually a bit uncomfortable with "InApp" because the "InSafari" is somewhat still _in app_. =P

I'll get rid of the articles. =)